packages/ui-components/src/Common/BaseCodeBox/index.tsx

```diff
@@ -81,6 +83,7 @@ const BaseCodeBox: FC<PropsWithChildren<CodeBoxProps>> = ({
   buttonText,
   as = 'a',
   showCopyButton = true,
+  copied = false,
```

Comment:
I don't think you should add this prop, you can change the `buttonText` prop to be of type ReactNode and pass the icon directly **with** the text.